ebu_tt_live/documents/ebuttd.py

```diff
@@ -3,16 +3,16 @@
 from .base import SubtitleDocument, TimeBase
 from ebu_tt_live import bindings
 from ebu_tt_live.bindings.converters.ebutt3_ebuttd import EBUTT3EBUTTDConverter
-
-
+from ebu_tt_live.documents.ebutt3 import TimelineUtilMixin
```

Comment:
Importing from `ebutt3` into `ebuttd` isn't a good look, and suggests this functionality should be in a common shared place. Please could you move the timing util objects into a separate file called something like `time_utils.py` in the same `documents` folder and import it from both here and `ebutt3.py`?